plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -35,10 +35,23 @@ import (
 )
 
 const (
+	bucketRange        = `(0,1,2,3,4,5,6,7,8,9)`
 	querySpanByTraceID = `
-		SELECT trace_id, span_id, parent_id, operation_name, flags, start_time, duration, tags, logs, refs, process 
-		FROM traces 
+		SELECT trace_id, span_id, parent_id, operation_name, flags, start_time, duration, tags, logs, refs, process
+		FROM traces
 		WHERE trace_id = ?`
+
+	queryByTag                     = `SELECT trace_id FROM tag_index WHERE service_name = ? AND tag_key = ? AND tag_value = ? and start_time > ? and start_time < ? limit ?`
+	queryByServiceName             = `select trace_id from service_name_index where bucket in ` + bucketRange + ` and service_name = ? and start_time > ? and start_time < ? limit ?`
+	queryByServiceAndOperationName = `select trace_id from service_operation_index where service_name = ? and operation_name = ? and start_time > ? and start_time < ? limit ?`
+	queryByDuration                = `select trace_id from span_duration_index where bucket = ? and service_name = ? and span_name = ? and duration > ? and duration < ? limit ?`
```

Comment:
can we format queries the same as above, to be readable without horizontal scrolling?